### ISSUE TITLE
Add some docstrings 

### ISSUE THREAD

**Issue by chris-greening** [chris-greening/instascrape#13]
**Is your feature request related to a problem? Please describe.**
Add some docstrings to places they're missing to improve the documentation for this repo. 

**Describe the solution you'd like**
Docstrings in [numpy](https://numpydoc.readthedocs.io/en/latest/format.html) style

**Comment by abhisht51** [comment]
Hi, 
I went through the several python files and I noticed the docstrings were already present in most of them. Where exactly are do you want me to add docstrings? I don't suppose class methods / internal functions would require them. @chris-greening

**Comment by chris-greening** [comment]
Hey @abhisht51! Thanks for asking, quick fixes you could check out are 
- [landing_page.py](https://github.com/chris-greening/instascrape/blob/master/instascrape/landing_page.py)
- [login_and_signup.py](https://github.com/chris-greening/instascrape/blob/master/instascrape/login_and_signup.py) 
- [http_error_page.py](https://github.com/chris-greening/instascrape/blob/master/instascrape/http_error_page.py)
it looks like the docstrings on those were leftovers from copying and pasting from [profile.py](https://github.com/chris-greening/instascrape/blob/master/instascrape/profile.py) and I forgot to indicate they aren't Profile objects lol 

Otherwise you can take a look at jsontools.py, there are some methods in there that haven't been included in their respective classes Methods section of the docstring (i.e. JSONScraper.to_dict) 

Basically anywhere you think a docstring would improve the documentation or add something to a preexisting docstring that isn't already there (examples, better wording, better formatting, documenting public methods, etc) is fair game!

**Comment by Siddhant-K-code** [comment]
can you please, what is the docstring, in short, I am new in with it, it looks intresting

**Comment by chris-greening** [comment]
> can you please, what is the docstring, in short, I am new in with it, it looks intresting

They are used for documenting Python code! Check out [this](https://realpython.com/documenting-python-code/) link for more information :smile:
